github/ci/prow-deploy/files/jobs/RHsyseng/rhcos-slb/rhcos-slb-presubmits.yaml

```diff
@@ -0,0 +1,35 @@
+---
+presubmits:
+  RHsyseng/rhcos-slb:
+    - name: pull-rhcos-slb-ci-network-tests-main
+      branches:
+        - main
```

Comment:
No need to specify main in branches, leaving it undefined should be ok. 
```suggestion
```

Reply:
[Done](https://github.com/kubevirt/project-infra/compare/b4fcea23fba6cfdfa4c2034ea75a7c4c84618945..3caea8590b31a7254decbe7cd4d38641a8b9fab5)